step2-05/README.md

```diff
@@ -12,17 +12,19 @@ f(data) => view
 
 This is fine if the data never changes. However, React exists to deal with data updates via props (immutable) and state (changes based on `setState()` API). In the real world, data is shaped like a tree and view is shaped like a tree. They don't always match.
```

Comment:
Not quite sure what it means "data is shaped like a tree and view is shaped like a tree"

Reply:
Wow I feel so accomplished now :D